File: src/main/java/util/MessageUtil.java
package util;

import javax.faces.application.FacesMessage;
import javax.faces.context.FacesContext;

public class MessageUtil {
    public static void addErrorMessage(String message) {
        message = "Erro: " + message;

        FacesContext.getCurrentInstance().getExternalContext().getFlash().setKeepMessages(true);

        FacesContext.getCurrentInstance().addMessage(null,
                new FacesMessage(FacesMessage.SEVERITY_ERROR, "", message));
    }
}

File: src/main/java/util/MessageUtil.kt
package util

import javax.faces.application.FacesMessage
import javax.faces.context.FacesContext

object MessageUtil {
    fun addErrorMessage(message: String) {
        FacesContext.getCurrentInstance().externalContext.flash.isKeepMessages = true

        FacesContext.getCurrentInstance().addMessage(null,
                FacesMessage(FacesMessage.SEVERITY_ERROR, "", "Erro: $message"))
    }
}
